### PR TITLE
 Updated some DSO textures

### PR DESCRIPTION
**Pull request by sunshuwei**
Updated some DSO textures.

### CODE REVIEW

**Comment by sunshuwei** [comment]
This is the new NGC 1499, which is now part of the M42.
It looks more coordinated.
m42

**Comment by alex-w** [comment]
Thanks for notification! I'll check visual part of updated nebulae within star fields tonight, but at the moment just looking the photos given unrealistic visual view.

File: nebulae/default/textures.json [hunk 1]
```diff
@@ -1685,6 +1677,14 @@
 		"minResolution" : 2.1216,
 		"maxBrightness" : 16.0
 	},
+	{
+		"imageCredits" : {"short" : "Zhu Ying""},
```
Comment (alex-w) [inline nebulae/default/textures.json]: Critical typo

**Comment by alex-w** [comment]
New IC4592 is acceptable.

**Comment by alex-w** [comment]
Well, new M42 is acceptable too, but it's not very realistic by visual observations and good for photographic observations.

Please fix the typo.

**Comment by alex-w** [comment]
Are you was make a some postprocess for new M42?

**Comment by sunshuwei** [comment]
I have post-processing M42 several times,  but it's not too good.
Maybe NGC1999 could be replaced separately.

**Comment by alex-w** [comment]
Just for understanding: I may accept new M42, but I prefer to get some discussion for other developers and you. Could you propose changes around M42 in separate PR?

**Comment by sunshuwei** [comment]
Yes，use new PR

Maybe this is also good.

**Pull request merged**